### ISSUE TITLE
Limit how many people someone can endorse per day?

### ISSUE THREAD

**Issue by poonen** [roed314/seminars#225]
Should we put an upper bound on the number of email addresses an endorsed user can endorse per day?  The question is whether this would help keep the R_0 down, in case the site gets infected by one rogue endorsed user.

**Comment by AndrewVSutherland** [comment]
This seems like a good idea to me.  Maybe 3 endorsements per day?

**Comment by rbommel** [comment]
If we are afraid of spambots taking over, then we should also put some time lock maybe. Only allow one to endorse others after being endorsed <some time unit> yourself.

**Comment by poonen** [comment]
@AndrewVSutherland: I was thinking of a number more than 3, maybe something like 10 per day. Or maybe the number could start small and increase 24 hours after the user creates the user's first seminar or institution (since we won't really know whether the user is a reasonable one until after we see what content the user produces).

**Comment by AndrewVSutherland** [comment]
@rbommel This is already the case, only endorsed users can endorse others.

@poonen I like the idea of increasing it once the start adding content.

**Comment by rbommel** [comment]
@AndrewVSutherland Yes, but the problem is that this limit of 3 per day only makes sense if there is also a time lock for newly endorsed users. Otherwise, I can just endorse 3 copies of myself, endorsing 9 other copies of myself, and we still have exponential growth.

**Comment by AndrewVSutherland** [comment]
@rbommel Ah, now I understand your concern.  We could add a time lag for endorsements to take effect, or maybe only allow one endorsement prior to creating content.

**Comment by edgarcosta** [comment]
I think if we manage captcha, the registration page, we should be able to
limit the rate.
